store/sqlstore/channel_store.go

```diff
@@ -1259,20 +1261,42 @@ func (s SqlChannelStore) GetMembersForUser(teamId string, userId string) store.S
 
 func (s SqlChannelStore) SearchInTeam(teamId string, term string) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		searchQuery := `
+		queryFormat := `
 			SELECT
-			    *
+				*
 			FROM
-			    Channels
+				Channels
 			WHERE
-			    TeamId = :TeamId
+				TeamId = :TeamId
 				AND Type = 'O'
 				AND DeleteAt = 0
-			    SEARCH_CLAUSE
-			ORDER BY DisplayName
-			LIMIT 100`
+				%v
+			LIMIT 50`
```

Comment:
Why 50?

Reply:
We mentioned reducing the number of results returned in our discussion, and there didn't seem to be an opposition to the idea. 50 is still twice as many results as slack (and 5x Discord).